DQMOffline/Trigger/python/MuonMonitor_cff.py

```diff
@@ -0,0 +1,36 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.MuonMonitor_cfi import hltMuonmonitoring
+
+
+TrkMu16_DoubleTrkMu6NoFiltersNoVtx_monitoring = hltMuonmonitoring.clone()
+TrkMu16_DoubleTrkMu6NoFiltersNoVtx_monitoring.FolderName = cms.string('HLT/Muon/TrkMu16_DoubleTrkMu6NoFiltersNoVtx/')
+TrkMu16_DoubleTrkMu6NoFiltersNoVtx_monitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_TrkMu16_DoubleTrkMu6NoFiltersNoVtx_v*")
+TrkMu16_DoubleTrkMu6NoFiltersNoVtx_monitoring.muonSelection = cms.string("pt > 6 && eta<2.4")
+
+TrkMu17_DoubleTrkMu8NoFiltersNoVtx_monitoring = hltMuonmonitoring.clone()
+TrkMu17_DoubleTrkMu8NoFiltersNoVtx_monitoring.FolderName = cms.string('HLT/Muon/TrkMu17_DoubleTrkMu8NoFiltersNoVtx/')
+TrkMu17_DoubleTrkMu8NoFiltersNoVtx_monitoring.numGenericTriggerEventPSet.hltPaths = cms.string("HLT_TrkMu17_DoubleTrkMu8NoFiltersNoVtx_v*")
+TrkMu17_DoubleTrkMu8NoFiltersNoVtx_monitoring.muonSelection = cms.string("pt > 8 && eta<2.4")
+
+
+DST_DoubleMu3_noVtx_CaloScouting_monitoring = hltMuonmonitoring.clone()
+DST_DoubleMu3_noVtx_CaloScouting_monitoring.FolderName = cms.string('HLT/Muon/DST_DoubleMu3_noVtx_CaloScouting/')
+DST_DoubleMu3_noVtx_CaloScouting_monitoring.numGenericTriggerEventPSet.hltPaths = cms.string("DST_DoubleMu3_noVtx_CaloScouting_v*")
```

Comment:
In case it is useful, this is the JIRA for the scouting dimuon path : https://its.cern.ch/jira/browse/CMSHLT-1243